se/__init__.py

```diff
@@ -196,14 +196,14 @@ def print_error(message: Union[SeException, str], verbose: bool = False) -> None
 	# is rendered in blue
 	message = regex.sub(r"`(.+?)`", stylize(r"\1", fg("light_blue")), message)
 
-	print("{}{} {}".format(MESSAGE_INDENT if verbose else "", stylize(" Error ", bg("red") + attr("bold")), message), file=sys.stderr)
+	print(f"{MESSAGE_INDENT if verbose else ''}{stylize(' Error ', bg('red') + attr('bold'))} {sys.stderr}")
```

Comment:
In this line, it looks like you removed the `file=` argument. This causes the function to fail. Can you update this? `file=sys.stderr` is an argument to `print()`, not something to pass to the f-string.

Reply:
Yikes, embarassing! Thanks for catching that.